data/ship/cfg/TR1X_gameflow_demo_pc.json5

```diff
@@ -0,0 +1,280 @@
+{
+    // NOTE: bad changes to this file may result in crashes.
+    // Lines starting with double slashes are comments and are ignored.
+
+    // path to the main menu background image
+    "main_menu_picture": "data/images/title.webp",
+
+    // path to the savegame file
+    "savegame_fmt_legacy": "save_demo_pc.%d",
+    "savegame_fmt_bson": "save_demo_pc_%02d.dat",
+
+    "force_game_modes": null,
+    "force_save_crystals": null,
+    "demo_delay": 16,
+    "water_color": [0.45, 1.0, 1.0],
+    "draw_distance_fade": 22.0,
+    "draw_distance_max": 30.0,
+    "injections": [
+        "data/injections/backpack.bin",
+        "data/injections/braid.bin",
+        "data/injections/lara_animations.bin",
+        "data/injections/lara_jumping.bin",
+        "data/injections/uzi_sfx.bin",
+    ],
+    "convert_dropped_guns": false,
+
+    "levels": [
+        // Level 2
+        {
+            "title": "City of Vilcabamba",
+            "file": "data_demo_pc/level2.phd",
+            "type": "level_demo_pc",
+            "music": 0,
+            "injections": [
+                "data/injections/vilcabamba_itemrots.bin",
+                "data/injections/vilcabamba_textures.bin",
+            ],
+            "sequence": [
+                {"type": "loading_screen", "picture_path": "data/images/peru.webp", "display_time": 5},
+                {"type": "start_game"},
+                {"type": "loop_game"},
+                {"type": "stop_game"},
+                {"type": "level_stats", "level_id": 0},
+                {"type": "exit_to_level", "level_id": 1},
+            ],
+            "strings": {
+                "key1": "Silver Key",
+                "puzzle1": "Gold Idol"
+            },
+            "demo": true,
```

Comment:
Here were the final stats I got. So I think `"unobtainable_pickups": 5` and `"unobtainable_kills": 9`.
![image](https://github.com/LostArtefacts/TR1X/assets/81546780/41cab93e-b7a0-46f1-81b2-3f34540731d1)

We don't have an `unobtainable_secrets` gameflow option either since normally that doesn't come up.

Reply:
Made a quick PR #1380 to support this.